plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/importer/page/GitImporterPagePresenter.java

```diff
@@ -166,6 +177,14 @@ public void branchNameChanged(@NotNull String branch) {
     public void go(@NotNull AcceptsOneWidget container) {
         container.setWidget(view);
 
+        if (StringUtils.isNullOrEmpty(dataObject.getName()) && StringUtils.isNullOrEmpty(dataObject.getSource().getLocation())) {
```

Comment:
Use com.google.common.base.Strings#isNullOrEmpty and deprecate method StringUtils.isNullOrEmpty